drivers/dac/Kconfig.gd32

```diff
@@ -0,0 +1,15 @@
+# DAC configuration options
+
+# Copyright (c) 2020 BrainCo Inc.
```

Comment:
It's 2021 actually :)

Reply:
Sorry, I should check these before push it in here.

Reply:
done.